packages/DatePicker/src/shared.styles.js

```diff
@@ -0,0 +1,11 @@
+import tokens from "@paprika/tokens";
+
+export const HoveredItemStyles = `
+  background-color: ${tokens.color.blackLighten70};
+  font-weight: bold;
```

Comment:
I don't like the bold on hover.  I know it's probably that way in the mocks, but it makes the text shift a little.  Small detail, but my `GiveAFudge` scale is disproportionately high for this kind of thing.

Reply:
Talked with @sunminsarahlee , let's keep `bold` on hover.

Reply:
Okay.